lib/matplotlib/tests/test_collections.py

```diff
@@ -1336,3 +1336,24 @@ def test_striped_lines(fig_test, fig_ref, gapcolor):
     for x, gcol, ls in zip(x, itertools.cycle(gapcolor),
                            itertools.cycle(linestyles)):
         ax_ref.axvline(x, 0, 1, linestyle=ls, gapcolor=gcol, alpha=0.5)
+
+
+@check_figures_equal(extensions=['png', 'pdf', 'svg', 'eps'])
+def test_hatch_linewidth(fig_test, fig_ref):
```

Comment:
This test does not seem correct.  Looking at what this code draws there is nothing shown (I think all of the circles are drawn at 0, 0 in screen space and clipped to not be visible by the axes).